test/tests/fips_test_suite/common.rs

```diff
@@ -2,7 +2,107 @@
 
 use caliptra_builder::firmware::{self, APP_WITH_UART, FMC_WITH_UART};
 use caliptra_builder::ImageOptions;
-use caliptra_hw_model::{BootParams, DefaultHwModel, HwModel};
+use caliptra_common::mailbox_api::*;
+use caliptra_hw_model::{BootParams, DefaultHwModel, HwModel, ModelError};
+use zerocopy::{AsBytes, FromBytes};
+
+// =================================
+//       EXPECTED CONSTANTS
+// =================================
+
+// Constants are grouped into RTL, ROM, and Runtime
+// Values can be specified for specific release versions (i.e. 1.0.1)
+// The user can specify which release (or default to current) to use when executing tests
+// Subsequent versions should "inherit" the previous version and override any changed values
+// The "current" struct must always match the behavior of components built from the same commit ID
+
+// ===  RTL  ===
+pub struct HwExpVals {
+    pub hw_revision: u32,
+}
+
+const HW_EXP_1_0_0: HwExpVals = HwExpVals { hw_revision: 0x1 };
+
+const HW_EXP_CURRENT: HwExpVals = HwExpVals { ..HW_EXP_1_0_0 };
+
+// ===  ROM  ===
+pub struct RomExpVals {
+    pub rom_version: u16,
+    pub capabilities: [u8; 16],
+}
+
+const ROM_EXP_1_0_1: RomExpVals = RomExpVals {
+    rom_version: 0x801, // 1.0.1
+    capabilities: [
+        0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1,
+    ],
+};
+const ROM_EXP_CURRENT: RomExpVals = RomExpVals { ..ROM_EXP_1_0_1 };
+
+// ===  RUNTIME  ===
+pub struct RtExpVals {
+    pub fmc_version: u16,
+    pub fw_version: u32,
+}
+
+const RT_EXP_CURRENT: RtExpVals = RtExpVals {
+    fmc_version: 0x0,
+    fw_version: 0x0,
+};
+
+// === Getter implementations ===
```

Comment:
Please update the README for the test suite with how to use these variables.

Reply:
Yep I'll definitely be adding documentation for all those options